src/core/tests/eval.cpp

```diff
@@ -2655,6 +2656,1216 @@ TEST(eval, evaluate_fake_quantize_dynamic_input) {
                 Pointwise(FloatEq(), std::vector<float>{2.f, 2.6666667f, 2.6666667f, 3.3333333f, 3.3333333f, 4.f}));
 }
 
+///////////////////////////////////// FakeConvert (FP8)
+
+namespace testing {
+namespace fp8 {
+constexpr float MAX_F8E4M3 = 448.f;
+constexpr float MIN_F8E4M3 = 0.001953125f;
+
+constexpr float MAX_F8E5M2 = 57344.f;
+constexpr float MIN_F8E5M2 = 0.0000152587890625f;
+}  // namespace fp8
+}  // namespace testing
+
+///////////////////////////////////// FakeConvert f8e5m2
+
+TEST(eval, evaluate_fake_convert_f32_to_f8e4m3_scale_small) {
+    using namespace testing;
+    constexpr auto et = element::f32;
+
+    std::vector<float> input_data{fp8::MIN_F8E4M3, fp8::MIN_F8E4M3 / 2.f, fp8::MIN_F8E4M3 / 4.f};
+
+    const auto data_shape = Shape{input_data.size()};
+
+    auto data = make_shared<ov::op::v0::Parameter>(et, data_shape);
+    auto max_input_val = std::abs(*std::max_element(input_data.begin(), input_data.end()));
+
+    auto scale = op::v0::Constant::create(et, Shape{1}, {fp8::MAX_F8E4M3 / max_input_val});
+    auto shift = op::v0::Constant::create(et, Shape{1}, {0.f});
+
+    auto op = make_shared<op::v13::FakeConvert>(data, scale, shift, "f8e4m3");
+
+    auto model = make_shared<Model>(OutputVector{op}, ParameterVector{data});
+
+    auto result = ov::Tensor();
+    auto out_vector = ov::TensorVector{result};
+    auto in_vector = ov::TensorVector{make_tensor<et>(data_shape, input_data)};
+    ASSERT_TRUE(model->evaluate(out_vector, in_vector));
+    result = out_vector.at(0);
+
+    EXPECT_EQ(result.get_element_type(), et);
+    EXPECT_EQ(result.get_shape(), data_shape);
+    EXPECT_THAT(read_vector<float>(result),
+                Pointwise(FloatEq(), std::vector<float>{0.001953125f, 0.0009765625f, 0.00048828125f}));
+}
+
+TEST(eval, evaluate_fake_convert_f32_to_f8e4m3_scale_1_small) {
+    using namespace testing;
+    constexpr auto et = element::f32;
+
+    std::vector<float> input_data{fp8::MIN_F8E4M3, fp8::MIN_F8E4M3 / 2.f, fp8::MIN_F8E4M3 / 4.f};
+
+    const auto data_shape = Shape{input_data.size()};
+    auto data = make_shared<ov::op::v0::Parameter>(et, data_shape);
+    auto scale = op::v0::Constant::create(et, Shape{1}, {1.f});
+    auto shift = op::v0::Constant::create(et, Shape{1}, {0.f});
+
+    auto op = make_shared<op::v13::FakeConvert>(data, scale, shift, "f8e4m3");
+    auto model = make_shared<Model>(OutputVector{op}, ParameterVector{data});
+
+    auto result = ov::Tensor();
+    auto out_vector = ov::TensorVector{result};
+    auto in_vector = ov::TensorVector{make_tensor<et>(data_shape, input_data)};
+    ASSERT_TRUE(model->evaluate(out_vector, in_vector));
+    result = out_vector.at(0);
+
+    EXPECT_EQ(result.get_element_type(), et);
+    EXPECT_EQ(result.get_shape(), data_shape);
+    EXPECT_THAT(read_vector<float>(result), Pointwise(FloatEq(), std::vector<float>{0.001953125f, 0.f, 0.f}));
+}
+
+TEST(eval, evaluate_fake_convert_f32_to_f8e4m3_scale_big) {
+    using namespace testing;
```

Comment:
maybe instead doing it in test globaly pull what is required like
`using testing::Pointwise` etc

Reply:
Actually the testing namespace is used for the whole file, anyway the tests are going to be moved to the "template" tests.